full-service/migrations/2021-03-31-220723_nullable_transaction_log_account_id/up.sql

```diff
@@ -0,0 +1,26 @@
+-- ALTER TABLE transaction_logs ALTER COLUMN account_id_hex DROP NOT NULL;
```

Comment:
Note: for foreign key constraints, we need to have nullable fields. The downside is that there are now "two ways" for a string to be "null" (either null or empty string), but the benefit of having proper FK constraints is worth it.

Reply:
The empty string will never pass foreign key constraints, so this is not a problem.